.buildkite/packaging.pipeline.yml

```diff
@@ -172,6 +174,73 @@ steps:
         artifact_paths:
           - build/distributions/**/*
 
+      - label: "SNAPSHOT: {{matrix}} FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_FIPS}"
+          SNAPSHOT: true
+          # packaging with `DEV=true` may cause linker issues while crosscompiling https://github.com/elastic/beats/issues/41270
+          DEV: false
+          FIPS: true
+        command: ".buildkite/scripts/packaging/package-dra.sh {{matrix}}"
+        agents:
+          provider: gcp
+          image: "${IMAGE_UBUNTU_X86_64}"
+          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+        timeout_in_minutes: 40
+        retry:
+          automatic:
+            - limit: 1
+        artifact_paths:
+          - build/distributions/**/*
+        matrix:
+          - x-pack/auditbeat
+          - x-pack/filebeat
+          - x-pack/metricbeat
```

Comment:
I believe this should also contain `x-pack/cloudbeat`

Reply:
We don't build cloudbeat in the beats repository, that lives in https://github.com/elastic/cloudbeat